src/address/types.rs

```diff
@@ -237,6 +237,8 @@ pub enum AddressError {
     InvalidUser,
     /// Invalid email domain
     InvalidDomain,
+    /// Fallback
+    Invalid(Vec<String>),
```

Comment:
In order not to break everything I added a variant here for the `chumsky` parsing error. If you have a better idea let me know!

Reply:
This is technically breaking, I think it is probably fine but if we are going to make a breaking change we should probably add `#[non_exhaustive]` while we are at it to allow us to do this again in the future without another breaking change.

Reply:
:heavy_check_mark:

Reply:
Let's leave this open for @ @paolobarbolini to explicitly ok.

Reply:
I think this is fine, there are other open PRs that might end up adding breaking changes so there will be a 0.11 soon.